tests/e2e/utils/lib.js

```diff
@@ -1,12 +1,5 @@
-export const waitForSelector = async function ( page, selector ) {
-	return await page.evaluate(
-		( selector ) => Boolean( document.querySelector( selector ) ),
-		selector
-	);
-};
-
 export const waitForElementCount = async function ( page, domSelector, count ) {
-	return await page.waitForFunction(
+	return page.waitForFunction(
```

Comment:
`return await` is an anti-pattern that eslint would normally catch. I've removed it.